test/object-store/primitive_list.cpp

```diff
@@ -808,24 +808,32 @@ TEMPLATE_TEST_CASE("primitive list", "[primitives]", cf::MixedVal, cf::Int, cf::
         if (!util::EventLoop::has_implementation())
             return;
 
-        SyncServer server;
-        SyncTestFile sync_config(server, "shared");
-        sync_config.schema = config.schema;
+        TestSyncManager init_sync_manager({}, {false});
+        auto& server = init_sync_manager.sync_server();
+        SyncTestFile sync_config(init_sync_manager, "shared");
+        sync_config.schema = Schema{
+            {"object",
+             {{"value", PropertyType::Array | TestType::property_type},
+              {"_id", PropertyType::Int, Property::IsPrimary{true}}}},
+        };
         sync_config.schema_version = 0;
+        server.start();
```

Comment:
I don't see any reason to be delaying starting the server here?

Reply:
I don't think this test has been run for a long time. I just tried to make it work, but did not really succeed. The test is still not run.